README.md

```diff
@@ -154,14 +158,30 @@ $ limactl start
 $ lima nerdctl run -d --name nginx -p 127.0.0.1:8080:80 nginx:alpine
 ```
 
-### FreeBSD
+### Windows
 
-See [`./docs/freebsd.md`](docs/freebsd.md).
+Currently, there is experimental support for Windows (see below for features that are currently known to work).
 
-### Windows
+To set up on Windows, first make sure you have [installed `containerd`](https://github.com/containerd/containerd/blob/main/docs/getting-started.md#installing-containerd-on-windows) then run the following in PowerShell:
+
+```powershell
+$Version="1.2.1" # update to your version of choice
+# the following is based on amd64, for other architectures, see release page
+curl.exe -L https://github.com/containerd/nerdctl/releases/download/v$Version/nerdctl-$Version-windows-amd64.tar.gz -o nerdctl-$Version-windows-amd64.tar.gz
+tar.exe xvf nerdctl-$Version-windows-amd64.tar.gz
+Copy-Item .\nerdctl.exe $Env:ProgramFiles\containerd -Force
+```
+If you already added `$Env:ProgramFiles\containerd` in the `PATH`, you should have `nerdctl` accessible from anywhere now:
 
-- Linux containers: Known to work on WSL2
-- Windows containers: experimental support for Windows (see below for features that are currently known to work)
+```powershell
+nerdctl --version
+```
+
+> For Linux Containers on Windows (LCOW), you can use WSL2 and follow similar steps for [Linux](#linux).
```

Comment:
To me, LCOW means running Linux containers on a Windows executable of `containerd.exe`.
nerdctl doesn't support this LCOW.

Reply:
Ooh, yes, this mixes up stuff. I'll revert back to the previous NOTE instead.